### PR TITLE
add copy over for insert, temporary loading message

### PR DESCRIPTION
**Pull request by mahos**
- date/datetime/time insert input fields still need extra format conversion for correct copy over

### CODE REVIEW

**Review by guzman-raphael — APPROVED** [review]
Good to proceed from my side but I'll give @Synicix a chance to review as well.

File: src/Components/MainTableView/TableView.tsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ type TableViewState = {
   tableContentData: Array<any>,
   tableInfoData: string,
   selectedTable: string,
-  errorMessage: string
+  errorMessage: string,
+  isFetchingTuples: boolean
```
Comment (Synicix) [inline src/Components/MainTableView/TableView.tsx:21]: Perhaps change this to a more generic name such as isReady to account for other loading things that might need to happen.

**Review by Synicix — CHANGES_REQUESTED** [review]
One minor change.

Also I filed #48 as a todo to correspond with the missing functionailty

**Comment by Synicix** [comment]
@mahos

File: src/Components/MainTableView/TableView.tsx (hunk 1 above)
Comment (mahos) [reply src/Components/MainTableView/TableView.tsx]: changed to isLoading

**Review by Synicix — APPROVED** [review]

**Comment by Synicix** [comment]
Hmm I don't have permission to merge..

**Pull request merged**